### PR TITLE
Sync image

### LINKED ISSUE #60: [EdgeX Auto-Collector] Synchronize the images of edgex to openyurt
**What would you like to be added**:
When we collect the edgex configuration, we need to pull the image according to the image field of each component and push it to the repo of openyurt.
All of the images have been collected in the Image field of `Component`. You may want to add a image synchronization feature to `Version` or `Component` and call it when appropriate.

**Why is this needed**:
Refer to [EdgeX Auto-colllector Proposal](https://github.com/openyurtio/openyurt/blob/master/docs/proposals/20221109-edgex-auto-collector.md#images-management)

**others**
/kind feature

### PR DESCRIPTION
**Pull request by yangbobo6**
#### What this PR does / why we need it:
Merge the crawled images into single-architecture and multi-architecture, and push to openyurt

#### Which issue(s) this PR fixes:
Fixes #60

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,6 +8,7 @@ require (
 	github.com/onsi/gomega v1.19.0
 	github.com/openyurtio/api v0.0.0-20220907024010-e5bfc9cc1b4b
 	github.com/pkg/errors v0.9.1
+	github.com/sirupsen/logrus v1.9.0
```
Comment (LavenderQAQ) [inline go.mod:11]: `yurt-edgex-manager` does not use `logrus` library. Try to delete it, then run `go mod tidy`.

**Comment by LavenderQAQ** [comment]
Another problem is that we need to replace the original image repo prefix in the configuration file with openyurt instead.
Example:
```yaml
- name: edgex-device-rest
  image: edgexfoundry/device-rest:2.3.0
  volumns:
    - name: edgex-init
      hostPath: /edgex-init
      mountPath: /edgex-init
    - name: anonymous_volume1
      hostPath: /tmp/edgex/secrets/device-rest
      mountPath: /tmp/edgex/secrets/device-rest
  ports:
    - protocol: TCP
      port: 59986
      targetPort: 59986
  componentEnv:
    SERVICE_HOST: edgex-device-rest
```
Replace to:
```yaml
- name: edgex-device-rest
  image: openyurt/device-rest:2.3.0
  volumns:
    - name: edgex-init
      hostPath: /edgex-init
      mountPath: /edgex-init
    - name: anonymous_volume1
      hostPath: /tmp/edgex/secrets/device-rest
      mountPath: /tmp/edgex/secrets/device-rest
  ports:
    - protocol: TCP
      port: 59986
      targetPort: 59986
  componentEnv:
    SERVICE_HOST: edgex-device-rest
```

**Comment by LavenderQAQ** [comment]
You can pass the `REPO` environment variable from the makefile into the collector as a flag
```makefile
.PHONY: run
run:
	go run main.go --debug=$(DEBUG) --repo=$(REPO)
```

**Comment by yangbobo6** [comment]
> You can pass the `REPO` environment variable from the makefile into the collector as a flag
> 
> ```makefile
> .PHONY: run
> run:
> 	go run main.go --debug=$(DEBUG) --repo=$(REPO)
> ```

I'm going to fix these problem right away.

File: tools/collector/edgex/collect.go [hunk 2]
```diff
@@ -17,13 +17,20 @@ limitations under the License.
...
 	"github.com/sirupsen/logrus"
 )
 
 var (
 	collectLog           *logrus.Entry
 	branchesURL          = "https://github.com/edgexfoundry/edgex-compose/branches/all"
 	extractVersionRegexp = `branch="(.*?)"`
+	singleArchPath       = "./config/singlearch_imagelist.txt"
+	multiArchPath        = "./config/multiarch_imagelist.txt"
+	repoName             = "openyurt/"
```
Comment (LavenderQAQ) [inline tools/collector/edgex/collect.go:33]: The repo name should be read from the makefile to make debugging easier.

**Review by LavenderQAQ — CHANGES_REQUESTED** [review]

File: tools/collector/edgex/collect.go (hunk 2 above)
Comment (yangbobo6) [reply tools/collector/edgex/collect.go:33]: ok,I will change it.

**Review by LavenderQAQ — APPROVED** [review]

**Pull request merged**